tests/unit/dataactbroker/test_submission_handler.py

```diff
@@ -0,0 +1,130 @@
+import pytest
+
+import datetime
+
+from dataactbroker.handlers.submission_handler import get_submission_metadata
+
+from dataactcore.models.lookups import PUBLISH_STATUS_DICT
+from dataactcore.models.jobModels import FileType, JobStatus, JobType
+
+from tests.unit.dataactcore.factories.domain import CGACFactory, FRECFactory
+from tests.unit.dataactcore.factories.job import SubmissionFactory, JobFactory, CertifyHistoryFactory
+from tests.unit.dataactcore.factories.staging import DetachedAwardFinancialAssistanceFactory
+
+
+@pytest.mark.usefixtures("job_constants")
+def test_get_submission_metadata(database):
+    """ Tests the get_submission_metadata function """
+    sess = database.session
+
+    now = datetime.datetime.utcnow()
+    now_plus_10 = now + datetime.timedelta(minutes=10)
+    start_date = datetime.date(2000, 1, 1)
+    cgac = CGACFactory(cgac_code='001', agency_name='CGAC Agency')
+    frec_cgac = CGACFactory(cgac_code='999', agency_name='FREC CGAC')
+    frec = FRECFactory(frec_code='0001', agency_name='FREC Agency', cgac=frec_cgac)
+    # Quarterly DABS cgac corrected
+    sub_1 = SubmissionFactory(submission_id='1', created_at=now, updated_at=now_plus_10, cgac_code=cgac.cgac_code,
```

Comment:
Should we separate each type of submission into its own test?

Reply:
I was partially keeping them in the same test so I could show we were getting the right one, but we can break them up